.github/ISSUE_TEMPLATE.md

```diff
@@ -1,17 +1,9 @@
-### Describe the problem:
+<!-- Provide a general summary of your changes in the Title above -->
 
-  * With couple of sentences describe the problem and its context.
+### Description:
+<!-- describe the problem or a feature request and its context -->
 
 #### Steps to reproduce:
-
-  1. 
-  2. 
-  3. 
-  
-#### Observed Results:
-
-  * What happened?  This could be a description, log output, etc.
-  
-#### Expected Results:
-
-  * What did you expect to happen?
```

Comment:
i'm actually ok with leaving the "observed results" and "expected results" in

Reply:
yeah, I wanted to make it simpler and the issue with GH "issues" is that you can use them also for capturing enhancements, changes to test infrastructure, docs, etc. and in that case the observed and expected results doesn't make any sense. The only case it makes a sense is actually reporting a bug

> one question, should we have the travis update in this as well? (seems out of place)

yeah, it's unrelated, the travis job was failing because it didn't switch the namespace properly, so I've added this 200ms wait

Reply:
cool, it all sounds good to me